nimble_build_system/cad/shelf.py

```diff
@@ -403,7 +401,7 @@ def generate_shelf_model(self):
         return self._shelf_model
 
 
-    def generate_assembly_model(self, explode=False):
+    def generate_assembly_model(self, render_options=None):
```

Comment:
Would be good to split this function up rather than mute warnings about too many statements and branches. I am happy to do this on the next PR if you want?

Reply:
Yes, let's do that. It seems like a more efficient use of our time.